new tool there

globus1987 opened the pull request.

globus1987 committed:
new tool there

jaccz commented:
Thank you for adding the new tool to the list!
The list is not ordered alphabetically but chronologically. Please add the new tool at the end of the list.

globus1987 committed:
Update tools.md

added to the end plus cleand all entries which does not point to existing tools.

globus1987 commented:
update plus removed all non-existing tools. the list was checked several times
